lib/controllers/users.js

```diff
@@ -178,4 +178,14 @@ module.exports.post = {
     }
 };
 
+module.exports.credentialSchema = joi.object().keys({
```

Comment:
As mentioned above, this shouldn't be defined in the user controller: it should be defined in the keys route or controller. 

Also, this schema should now contain a `user` property, which is a full `user` object.